content/akeneo-web-api.yaml

```diff
@@ -1604,6 +1671,33 @@ paths:
           $ref: "#/responses/403Error"
         406:
           $ref: "#/responses/406Error"
+    post:
```

Comment:
this endpoint has been developped in in 2.0.3, no?

Reply:
Hum yes but i think the branch patch-2.0.4 is not up do date with the 2.0.3 as I rebased it I think it comes from the 2.0.3 (check the commits)